source/templates/yml.slim

```diff
@@ -1,4 +1,10 @@
 .tab-yml.menu-content-padding[ng-controller="YMLController as ymlCtrl"]
+	section.cardo.warn[ng-if="appService.validationWarnings.length!=0"]
+		.icon
+		.message
+			.title == data[:strings][:yml][:warn][:title]
+			ul
+				li[ng-repeat="warning in appService.validationWarnings"] {{warning}}
```

Comment:
please change it to: li[ng-repeat="warning in appService.validationWarnings" ng-bind="warning"]